start-page/style.css

```diff
@@ -56,8 +56,12 @@ body, html {
     overflow: overlay;
 }
 #shortcutscontainer1-nonium {
+    scrollbar-color: white rgba(255, 255, 255, 0.6);
```

Comment:
This is just to match the Feren OS styling since the Chromium-based styling for it looks crud in Firefox.